15-Appendix.Rmd

```diff
@@ -0,0 +1,144 @@
+# (APPENDIX) Appendix {-}
+
+# Conda Package Management
+
+## Unified package management for both R and Python
+
+### Faster builds
+
+CRAN[^what_is_cran] mirrors do not provide linux[^linux_note] compiled binaries
+for packages. This means a long wait when doing `install.packages` both in an R
+Studio session and when running a Docker build for a `shiny` application.
+
+### Cross Language
+
+Both `dbtools` and `s3tools` rely on Python packages through the `reticulate` R
+to Python bridge. This means that `packrat` is not enough to reproducibly and
```

Comment:
Maybe: `packrat` only handles R dependencies, this means that it is not enought to ....